pkg/plugin/vector/embed/openai.go

```diff
@@ -86,12 +104,22 @@ func (o *openAIClient) Embed(ctx context.Context, model string, payload string)
 	return body.Data[0].Embedding, nil
 }
 
+func (o *openAIClient) Health(ctx context.Context, model string) error {
+	_, err := o.Embed(ctx, model, "Hello, world!")
```

Comment:
This part is still failing, when I log it it seems the model passed here is empty when hitting Save & test on a fresh install.
```
Vector service health check failed.
Error: vector service health check failed: embedder health: got non-2xx status from OpenAI
```

This seems like a front-end issue though. If I change the model used (e.g. `BAAI/bge-base-en-v1.5`) and change it back to the original (e.g. `BAAI/bge-small-en-v1.5`) it does work